src/sql/workbench/browser/designer/designer.ts

```diff
@@ -551,6 +553,9 @@ export class Designer extends Disposable implements IThemable {
 						this.handleEdit({ type: DesignerEditType.Update, path: propertyPath, value: args.value });
 					}
 				});
+				input.onInputFocus(() => {
+					this._propertiesPane.setDescriptionText(showDescription, componentDefinition);
```

Comment:
showDescription shouldn't be a parameter for the setDescriptionText method.

Reply:
Do you not want the descriptions to be updated as we click different options which might not have any descriptions?